prow/cmd/config-bootstrapper/README.md

```diff
@@ -6,4 +6,6 @@ config-updater Prow plugin.
 When a set of configurations do not exist (for example, on a clean redeployment or in a disaster
 recovery situation), the config-updater plugin is not useful as it can only upload incremental
 updates. This tool is meant to be used in those situations to set up the config to the correct
-base state and hand off ownership to the plugin for updates.
\ No newline at end of file
+base state and hand off ownership to the plugin for updates.
+
+`config-bootstrapper` uses the [`updateconfig`](https://github.com/kubernetes/test-infra/tree/master/prow/plugins/updateconfig) package to update the configmaps and uses configurations from `plugins.yaml` and `config.yaml` that are specified as command line arguments to it.
```

Comment:
I think for a user who just wants their config to work, specifying which package in code is used to run the tool is a little too low-level. Maybe we could write:

> Provide the `config-bootstrapper` with the latest state of the Prow configuration (`plugins.yaml`, `config.yaml`, any job configuration files) to boot-strap with the latest configuration.

Reply:
I added the changes with a usage example, let me know if I need to make any other changes.